Automattic/wp-calypso
# Akismet pricing: Upgrade calypso upgrade map

Related to #

## Proposed Changes

* Replaced the upgrade map with the new plans. 

## Testing Instructions



- Apply the branch locally and spin yarn start OR use Live link provided below
- Enable the store sandbox. 
- Navigate to checkout/akismet/ak_pro5h_yearly and buy the yearly plan
- Navigate to /me/purchases and select the new Purchased Akismet plan. You will see the upgrade button. Clicking on it 
will take you to the checkout/akismet/ak_bus5k_yearly plan. Purchase it. 

<img width="1250" alt="Screenshot 2023-11-02 at 18 45 44" src="https://github.com/Automattic/wp-calypso/assets/6594561/de84cf94-e185-4e95-9560-3fcf80a34559">

- Now, upgrade the business plan, which will take you to buy the ak_ep15k_yearly plan
- Purchase it and try to upgrade the ep15k plan, which will take you to the ep25k plan, and if you try to upgrade that it will you take to the https://akismet.com/enterprise/ page. 


## Pre-merge Checklist
